### PR TITLE
Metal3 quickstart updates

### PR DESCRIPTION
**Pull request by hardys**
- Adds an architecture overview diagram
- Updates image/chart locations
- Adds some downstream cluster manifest examples
- Completes some of the other TODO sections

Fixes: #178
Partially Fixes: #180
Fixes: #181

/cc @agracey I still have further work to do on this quickstart, but wanted to push this iteration for review, I'll leave it up to you if you'd rather merge then iterate on follow-ups, or treat this as a WIP pending further updates

### CODE REVIEW

File: asciidoc/quickstart/metal3.adoc [hunk 1]
```diff
@@ -75,9 +73,9 @@ This IP must be part of the controlplane subnet, and reserved for static configu
 +
 [,bash]
 ----
-helm repo add suse-edge https://suse-edge.github.io/charts
+helm pull oci://registry.suse.com/edge/metallb-chart
```
Comment (atanasdinov) [inline asciidoc/quickstart/metal3.adoc:76]: Can't we get rid of `helm pull` calls?

File: asciidoc/quickstart/metal3.adoc [hunk 2]
```diff
@@ -230,19 +229,253 @@ be provisioned, then bootstrapped to form an RKE2 cluster.
...
+                [Service]
+                Type=oneshot
+                User=root
+                ExecStartPre=/bin/sh -c "mount -L config-2 /mnt"
+                ExecStart=/bin/sh -c "sed -i \"s/BAREMETALHOST_UUID/$(jq -r .uuid /mnt/openstack/latest/meta_data.json)/\" /etc/rancher/rke2/config.yaml"
+                ExecStart=/bin/sh -c "echo \"node-name: $(jq -r .name /mnt/openstack/latest/meta_data.json)\" >> /etc/rancher/rke2/config.yaml"
+                ExecStartPost=/bin/sh -c "umount /mnt"
+                [Install]
+                WantedBy=multi-user.target
+    version: v1.28.3+rke2r1
```
Comment (atanasdinov) [inline asciidoc/quickstart/metal3.adoc:314]: Can we use 1.28.8? Applies to all references.

File: asciidoc/quickstart/metal3.adoc [hunk 3]
```diff
@@ -230,19 +229,253 @@ be provisioned, then bootstrapped to form an RKE2 cluster.
...
+        object: machine
+----
 
 === Worker/Compute deployment
 
-TODO
+Similar to the controlplane we define a yaml manifest, which contains the following resources:
 
-== Next steps
+* MachineDeployment defines the number of replicas (hosts) and the bootstrap/infrastructure provider (in this case RKE/Metal3)
```
Comment (atanasdinov) [inline asciidoc/quickstart/metal3.adoc:356]:
```suggestion
* MachineDeployment defines the number of replicas (hosts) and the bootstrap/infrastructure provider (in this case RKE2/Metal3)
```

File: asciidoc/quickstart/metal3.adoc (hunk 2 above)
Comment (hardys) [reply asciidoc/quickstart/metal3.adoc]: Done, thanks!

File: asciidoc/quickstart/metal3.adoc (hunk 3 above)
Comment (hardys) [reply asciidoc/quickstart/metal3.adoc]: Fixed, thanks!

File: asciidoc/quickstart/metal3.adoc (hunk 1 above)
Comment (hardys) [reply asciidoc/quickstart/metal3.adoc]: Yes I guess we can, removed - thanks!

**Review by atanasdinov — APPROVED** [review]

**Comment by agracey** [comment]
Let's keep this open for now since there shouldn't be anyone else trying to write this as well

@dariavladykina as a heads up, this page is in flux.

**Comment by joesusecom** [comment]
I've made a few additional comments based on the current (as of April 23) published version.

**Review by agracey — APPROVED** [review]
Based on slack conversation, let's merge then edit

**Pull request merged**